news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,149 @@
+
+## Introduction
+
+As done in the past, there's a few main and new terms/ideas you'll need to understand.
+
+Star Rating is designed to be a measure to reflect how hard (or not) a given beatmap is. It does this through mathematically charting various values in a map's construction, such as how densely packed a given portion of the map is with hitobjects, and so forth. These are described as *skills* as a general term. For the purposes of the star ratings described here, morth1 selected three skills: *stamina*, *rhythm* and *colour*.
+
+For the new terminology hitting difficulty calculation, we refer to *peaks* as a new skill utilised, based on the old component of strain. 
+
+For the purposes of Star Rating calculation, *strain* within the *peaks* skill has become the number that approximates it. Switching from the previous system, where strain and individual skill values formed Star Rating, strain is now the only summative factor in this calculation. In this case, there are two relevant subtypes of strain; while normal strain decays over time, *note-based strain* decays with every note hit, which is a subtle, yet important distinction only used within the old rhythm system, which has not yet been reworked. In both cases strain is increased by each individual hitobject in the map; we'll refer to that increase as *object strain*.
+
+We will also be using terminology coined by the osu!taiko community to refer to how patterns are placed. **k** refers to kat (blue), **d** refers to don (red), coined as different 'colours'. Long streams that are of one colour are referred to as *monos*, while notes that are surrounded by brackets () refer to **1/6** spacing, and notes inside square brackets [] refer to **1/8** spacing. 
+
+If you would like to know more about the changes posted, or for more concise information on the some of the in development changes, head on over [the global osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit#). This is a working document, and will periodically be updated with the latest, (and greatest) developments.
+
+## An insight into the base of the new rework
+
+Previous rework efforts have aimed to tackle all three skills at once, which has made it difficult to collect meaningful and targetted feedback. As such, the osu!taiko pp committee opted for a modular approach, tackling new changes and reported issues one stage at a time, in order to develop more accurate solutions. In this osu!taiko star rating rework, we have re-evaluated the stamina and colour components of difficulty, with the capacity for changes following community feedback. In addition, the performance points system has been adjusted to bring it closer in line with community expectations. During this time, we have been and will continue to work on other difficulty components (both small- and large-scale) in preparation for future releases.
```

Comment:
I assume this line was a leftover
```suggestion
Previous rework efforts have aimed to tackle all three skills at once, which has made it difficult to collect meaningful and targetted feedback. As such, the osu!taiko pp committee opted for a modular approach, tackling new changes and reported issues one stage at a time, in order to develop more accurate solutions. In this osu!taiko star rating rework, we have re-evaluated the stamina and colour components of difficulty, with the capacity for changes following community feedback. In addition, the performance points system has been adjusted to bring it closer in line with community expectations. During this time, we have been and will continue to work on other difficulty components (both small and large-scale) in preparation for future releases.
```

Reply:
yeah that was double up century, rewritten and applied in 1fa2745